services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -101,7 +102,9 @@ public Response getMainPage() throws URISyntaxException, IOException, FreeMarker
                 if (auth != null) {
                     Locale locale = session.getContext().resolveLocale(auth.getUser());
                     map.put("locale", locale.toLanguageTag());
-                    map.put("msg", messagesToJsonString(theme.getMessages(locale)));
+                    Properties messages = theme.getMessages(locale);
+                    map.put("javaMessages", new MessageFormatterMethod(locale, messages));
```

Comment:
Where is javaMessages used? I couldn't see it being used anywhere. In either case I would use a name not including Java. Perhaps "advancedMsg"?